src/osp/universe/coordinates.h

```diff
@@ -31,6 +31,18 @@
 namespace osp::universe
 {
 
+inline Vector3g rotate_vector3g(Vector3g const in, Quaterniond const rot) noexcept
+{
+    return Vector3g(rot.transformVector(Vector3d(in)));
```

Comment:
i take it these can't be constexpr because Vector3g and Quaterniond aren't able ot be?

Reply:
transformVector specifically; constructors and accessors work in constexpr